import {} does not work with this package.

The error is:
`error TS2305: Module '"acebase"' has no exported member 'AceBase'.`
Code: 
`import { AceBase } from "acebase";`
A band-aid to this is changing the `index.d.ts` file from
```
export * from '../types;
to
export * from '../types/index.js';
```

package.json
```
{
  "name": "kogasa-ts",
  ...
  "license": "ISC",
  "type": "module",
  "dependencies": {
    "acebase": "^1.27.0",
    "discord.js": "^14.7.1",
    "dotenv": "^16.0.3",
    "openai": "^3.1.0"
  }
}
```

--- a/create-package-files
+++ b/create-package-files
@@ -43,7 +43,7 @@ JSON
 
 # Write typings to support Node16 module resolution 
 cat >dist/esm/index.d.ts <<TYPESCRIPT
-export * from '../types';
+export * from '../types/index.d.ts';
 TYPESCRIPT
 
 # Write example file for runkit

